anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,77 @@
+= Upgrading and Migrating Anypoint MQ Connector Connector to 4.x 
+
+Upgrade Anypoint Connector for Anypoint MQ (Anypoint MQ Connector) to version 4.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|3.x |4.x
+|===
+
+== Changes in This Release
+
+Anypoint MQ Connector 4.x includes support for the Connected Apps feature.
+
+The changes included in this version break backward compatibility.
+
+This release contains a new format for the *URL* field in the *Global Element Properties* page.
+
+* In Anypoint MQ Connector 3.x, the URL format was:
++
+`https://mq-us-east-1.anypoint.mulesoft.com/api/v1`
+* In Anypoint MQ Connector 4.x, the URL format includes the orgID and envID. For example:
++
+`https://mq-us-east-1.anypoint.mulesoft.com/api/v1/organizations/orgID/environments/envId`
```

Comment:
Can we add link to the page which describes how can they obtain new URL from UI here?
If you are already doing that just let me know where to look on beta site or share a screenshot?